src/main/resources/application.conf

```diff
@@ -182,6 +182,12 @@ ergo {
     # which are potentially wallet's. But to be sure, script execution is needed, which could be costly to do in a bulk.
     # So we check from a queue only one box per "scanningInterval".
     scanningInterval = 1s
+
+    # Perform blocks scanning when the wallet is locked (may consume additinal disk space)
+    postponedScanning = true
+
+    # Save used boxes (may consume additinal disk space) or delete them immediately
+    keepHistory = false
```

Comment:
Don't like the name. Let's rename to something like `keepSpentBoxes`. Also I propose to set it to true by default

Reply:
Spent boxes are not used for now, there is also no interface to view them.